src/PS1/golden_captures.md

```
@@ -73,6 +73,28 @@ export QTMESH_PS1_GOLDEN_SCENE_ID=retail-a
+```bash
+export QTMESH_PS1_TEST_BIOS=/path/scph1001.bin
+export QTMESH_PS1_GOLDEN_RETAIL_C_ISO="/path/Crash Bandicoot - Warped (USA).cue"
+./build/bin/UnitTests --gtest_filter='MeshReconstructorGoldenTest.RetailCInCoreGoldenPass'
+```
+
+It boots the disc, captures one frame through the real `EmuCore`/`RipperHooks`
+path, reconstructs with stats, asserts the pass bar (`tracked > 0`,
+`tracked+depth ≥ 50%`, `!slabLike`, has bounds), and prints a doc-ready line to
+stderr:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)